exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SqlConverter.java

```diff
@@ -161,6 +161,19 @@ public SqlNode parse(String sql) {
       SqlParser parser = SqlParser.create(sql, parserConfig);
       return parser.parseStmt();
     } catch (SqlParseException e) {
+
+      // Attempt to use default back_tick quote character for identifiers, when QUOTING_IDENTIFIERS_CHARACTER
```

Comment:
is it really a desirable behavior? What's wrong with just returning an error if the quoting doesn't match what has been asked by the user?

A consequence of that change is that every invalid request will be parsed twice even if the issue has nothing to do with quoting.